@shared/api/internal.ts

```diff
@@ -687,6 +695,79 @@ export const getIndexerAccountHistory = async ({
   }
 };
 
+export const getAccountHistory = async (
+  publicKey: string,
+  networkDetails: NetworkDetails,
+) => {
+  if (isCustomNetwork(networkDetails)) {
+    return await getAccountHistoryStandalone({
+      publicKey,
+      networkDetails,
+    });
+  }
+  return await getIndexerAccountHistory({
+    publicKey,
+    networkDetails,
+  });
+};
+
+export const getIndexerTokenDetails = async ({
```

Comment:
should this be called `getTokenDetails1 because it handles both the standalone/custom case and the indexer case?

Reply:
yeah for sure it should 😄 , done in 797a2c5d99761ba9bb8948c7f11e8de8fd769c57